lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js

```diff
@@ -23,7 +23,7 @@ class InstructionSlide extends React.Component {
             <div className="container__row">
               <div className="container__block -narrow -centered">
                 <h2 className="heading -gamma">{title}</h2>
-                {competition.joined ? <p>{competition.messages.confirmation}</p> : ''}
+                {competition && competition.joined ? <p>{competition.messages.confirmation}</p> : ''}
```

Comment:
any chance we'd get `undefined` errors from `competition.messages.confirmation` not being present?